R/input.R

```diff
@@ -146,11 +146,11 @@ numericInput <- function(inputId, label, value, min = NA, max = NA, step = NA, w
 #' Change numeric input value and settings
 #'
 #' @param session The session object passed to function given to shinyServer.
-#' @param inputId The id of the input object.
+#' @param input_id The id of the input object.
 #' @param label The label to set for the input object.
 #' @param value The value to set for the input object.
 #' @param min Minimum value.
 #' @param max Maximum value.
 #' @param step Step size.
 #' @export
-updateNumericInput <- shiny::updateNumericInput
```

Comment:
I'd live this as well for consistency,

Reply:
are the parameters between shiny implementation and our inconsistent? if users wants to find docs for `updateNumericInput` then simple `?updateNumericInput` will explain it to him, I don't that would be important from what packages that comes

Reply:
I could agree, but this might be misleading to the users. They may not be aware of the fact that they can use raw shiny function. Wdyt?

Reply:
Don't have strong opinion here, so prob. you're right. Added back, back snake case also included.